pkg/storage/memory/memory.go

```diff
@@ -18,14 +18,37 @@ type (
 
 	storageMem struct {
 		sync.RWMutex
-		store map[string]memStorageSecret
+		storePruneTimer time.Ticker
+		store           map[string]memStorageSecret
```

Comment:
```suggestion
		store           map[string]memStorageSecret
		storePruneTimer time.Ticker
```

Just my OCD… 😄 [optional]

Reply:
Implemented and updated declaration to match.